site2/docs/reference-metrics.md

```diff
@@ -339,6 +341,31 @@ All the managed ledger bookie client metrics are labelled with the following lab
 | pulsar_managedLedger_client_bookkeeper_ml_workers_task_execution | Summary | The worker task execution latency calculated in milliseconds. |
 | pulsar_managedLedger_client_bookkeeper_ml_workers_task_queued | Summary | The worker task queued latency calculated in milliseconds. |
 
+### Token metrics
+
+All the token metrics are labelled with the following labels:
+
+- *cluster*: `cluster=${pulsar_cluster}`. `${pulsar_cluster}` is the cluster name that you have configured in the `broker.conf` file.
+
+| Name | Type | Description |
+|---|---|---|
+| pulsar_expired_token_count | Counter | The number of Pulsar expired tokens. |
```

Comment:
| pulsar_expired_token_count | Counter | The number of expired tokens in Pulsar. |